tests/jenkins/BuildPipelineTest.groovy

```diff
@@ -10,7 +10,8 @@ package jenkins.tests
 
 import org.junit.*
 import static com.lesfurets.jenkins.unit.global.lib.LibraryConfiguration.library
-import static com.lesfurets.jenkins.unit.global.lib.ProjectSource.projectSource
+// import static com.lesfurets.jenkins.unit.global.lib.ProjectSource.projectSource
```

Comment:
Shall we remove this comment then?

Reply:
yes, since we are not going back to local lib location at opensearch-build repo... I think it makes sense to remove it .. let me do that .. thanks